code/modules/mob/living/carbon/human/life.dm

```diff
@@ -171,7 +171,11 @@
 			eye_blind = 1
 			blinded = 1
 			eye_blurry = 1
-		else // if they're not supposed to have a vision organ, then they must see by some other means
+		else if(src.is_diona() && istype(organs_by_name["head"], /obj/item/organ/external/stump)) // if they're not supposed to have a vision organ, then they must see by some other means

```

Comment:
Why not just like, give dionae a vision organ and put it in the head. This is snowflakey.

Reply:
Because vision organ then will take damage, can be surgically removed, etc. We don't want that extra mechanics, because head is a whole nymph

Reply:
Thiiis smells like bad code and maintainability. You could set the `diona/vision_organ` to be head or something? Or `null`, alternatively.